core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -71,6 +71,7 @@ object ApiVersion {
     // and KafkaStorageException for fetch requests.
     KAFKA_1_1_IV0,
     // Introduced OffsetsForLeaderEpochRequest V1 via KIP-279
+    // Introduced OffsetCommitRequest V4 via KIP-211
```

Comment:
We're obviously not consistent about this, but we really only need to mention the inter-broker protocol changes here.